.travis.yml

```
@@ -53,7 +53,7 @@ jobs:
         - tests/scripts/test_psa_constant_names.py
         - tests/ssl-opt.sh
         # Modern OpenSSL does not support fixed ECDH or null ciphers.
```

```suggestion
        # Modern OpenSSL does not support fixed null ciphers.
```
Minor: need to modify comments.